pallets/grants/src/tests.rs

```diff
@@ -147,16 +147,25 @@ fn add_vesting_schedule_fails_if_zero_period_or_count() {
 
 #[test]
 fn add_vesting_schedule_fails_if_transfer_err() {
+	use sp_runtime::TokenError::FundsUnavailable;
 	ExtBuilder::default().one_hundred_for_alice().build().execute_with(|| {
 		let schedule = VestingSchedule {
 			start: 1u64,
 			period: 1u64,
 			period_count: 1u32,
-			per_period: 100u64,
+			per_period: 1u64,
```

Comment:
Do you need to change the per_period here?

Reply:
I don't remember this one. I will check